components/basecomponents/StoryText.tsx

```diff
@@ -54,21 +60,26 @@ const StoryText = ({ textLanguage, audioLanguage, id, audio, onClick }: StoryTex
 
     onClick(phraseInfo);
   };
+
+  if (!stories) return <div>Loading...</div>;
+
   return (
     <div className="mt-4 md:flex bg-slate-100 divide-y-8 divide-white md:divide-y-0 md:w-1/2">
       <div className="max-h-[30vh] md:max-h-full overflow-y-scroll md:overflow-auto" ref={containerRef}>
-        {STORIES[id].map((phrase: StoryPhrase, index: number) => (
-          <p
-            key={index}
-            onClick={handleClick}
-            ref={playing(phrase) ? phraseRef : null}
-            id={textLanguage === 'uk' ? phrase.start_uk.toString() : phrase.start_cs.toString()}
-            className={`hover:cursor-pointer mx-6 my-4 text-left ${playing(phrase) && 'text-[#013ABD]'} ${
-              played(phrase) && 'text-[#64a5da]'
-            }`}
-          >
-            {textLanguage === 'cs' ? phrase.main : phrase.uk}
-          </p>
+        {stories.map((phrase: StoryPhrase, index: number) => (
+          <div key={phrase.start_cs} className="flex w-full">
+            <p
+              key={index}
+              onClick={handleClick}
+              ref={playing(phrase) ? phraseRef : null}
+              id={textLanguage === 'uk' ? phrase.start_uk.toString() : phrase.start_cs.toString()}
```

Comment:
This is probably why phrase highlighting doesn't work

Reply:
fixed